device-base/Dockerfile.armv7hf.rpi.tpl

```diff
@@ -4,6 +4,13 @@ LABEL io.resin.device-type="#{DEV_TYPE}"
 
 RUN echo "deb http://archive.raspbian.org/raspbian #{SUITE} main contrib non-free rpi firmware" >>  /etc/apt/sources.list
 RUN apt-key adv --keyserver pgp.mit.edu  --recv-key 0x9165938D90FDDD2E
+RUN apt-get update && apt-get install -y --no-install-recommends wget \
+	&& wget http://archive.raspberrypi.org/debian/raspberrypi.gpg.key \
+	&& apt-key add raspberrypi.gpg.key \
+	&& rm -rf raspberrypi.gpg.key \
+	&& echo "deb http://archive.raspberrypi.org/debian/ jessie main" >> /etc/apt/sources.list \
```

Comment:
@petrosagg It comes from this thread https://gitter.im/resin-io/chat?at=57323356a351d8310952c20b. An user wanted to install some rpi specific packages but they are not available from this mirror http://archive.raspbian.org/raspbian jessie main contrib non-free rpi firmware. So I think we should add it to our raspberrypi base images